.storybook/preview.jsx

```diff
@@ -29,11 +33,21 @@ export const parameters = {
       date: /Date$/,
     },
   },
+  msw: {
+    handlers: [
+      // Only return mocked data when making requests to our mock stack
+      ...getHandlers(MOCK_REPO_ORIGIN),
```

Comment:
Register handlers on the MOCK_REPO_ORIGIN (which is `'https://mock-repo.sagebase.org'`)